main.go

```diff
@@ -109,3 +117,27 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+func cleanupPendingBuilds(curfs FileSystem) error {
+	l := log.New(os.Stderr, "[cleanup] ", log.LstdFlags)
+	projects, err := ioutil.ReadDir(cfg.ProjectsPath)
+	if err != nil {
+		return err
+	}
+
+	for _, f := range projects {
+		pendingPath := filepath.Join(cfg.BuildPath, f.Name(), "pending")
+		pendingBuilds, err := ioutil.ReadDir(pendingPath)
+		if len(pendingBuilds) > 0 {
+			for _, p := range pendingBuilds {
+				l.Printf("Removing pending build of %s project: %s", f.Name(), p.Name())
+				pendingBuildPath := filepath.Join(pendingPath, p.Name())
+				err = curfs.Remove(pendingBuildPath)
+				if err != nil {
+					return err
```

Comment:
We should return a more meaningful error here:

> Error pruning zombie build abc123 of project '%s'

Reply:
fixed